athenet/algorithm/utils.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+
+def list_of_percentage_rows_table(table, layer_id):
+    """returns list of tuples (percentage, number of row, [layer_id])
+       representing rows of [table]
+
+       "percentage" is a sum of absolute values in row divided by
+       sum of absolute values in all table
+
+       result list is sorted by "number of row"
+    """
+    sum_of_all = np.sum(abs(table))
+    result = []
+    for i in xrange(table.shape[0]):
+        result.append((np.sum(abs(table[i])) / sum_of_all, i, layer_id))
+    return result
+
+
+def list_of_percentage_columns(layer_id, layer):
+    return list_of_percentage_rows_table(np.transpose(layer.W), layer_id)
+
+
+def list_of_percentage_rows(layer_id, layer):
```

Comment:
is there any reason why sometimes first is layer, then layer_id and sometimes layer_id then layer? it was very confusing for me on first reading